erpnext/vehicles/report/vehicle_maintenance_reminder/vehicle_maintenance_reminder.json

```diff
@@ -0,0 +1,43 @@
+{
+ "add_total_row": 0,
+ "creation": "2022-11-23 14:34:05.388993",
+ "disable_prepared_report": 0,
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "idx": 0,
+ "is_standard": "Yes",
+ "letter_head": "Hyundai South",
```

Comment:
Remove `letter_head` from Report JSON
and maybe remove from other reports as well
and also how about we fix Report JSON exporter to not include `letter_head` in the exported file